### PR TITLE
fix: add missing line break to breaking changes file

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

 When creating the release I noticed that breaking changes were added with a missing line breaks. 
 This small single-line fix will add the required line break to enable us to automatically release.

### CODE REVIEW

File: tools/releaser/scripts/update-version.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ echo "Modifying $SDK_MAJOR_VERSION to $BUMPED_MAJOR_VERSION Resource Version acr
 npm exec -c "replace-in-file /$SDK_MAJOR_VERSION/g $BUMPED_MAJOR_VERSION $VERSION_UPDATE_PATHS --isRegex"
 
 echo "Creating empty breaking changes file for $BUMPED_MAJOR_VERSION"
-echo "# Breaking Changes" > "$script_path/../breaking_changes/${BUMPED_MAJOR_VERSION}.md"
+printf "\n# Breaking Changes\n" > "$script_path/../breaking_changes/${BUMPED_MAJOR_VERSION}.md"
```
Comment (gssbzn) [inline tools/releaser/scripts/update-version.sh:22]:
```suggestion
echo "# Breaking Changes\n" > "$script_path/../breaking_changes/${BUMPED_MAJOR_VERSION}.md"
```

Comment (wtrocki) [reply tools/releaser/scripts/update-version.sh:22]: shellcheck suggests printf :(

Comment (gssbzn) [reply tools/releaser/scripts/update-version.sh:22]: oh well, can you then just remove the opening `\n` I don't think is necessary

Comment (wtrocki) [reply tools/releaser/scripts/update-version.sh:22]: https://github.com/mongodb/atlas-sdk-go/releases/tag/v20230201008.0.0 seems to be missing `\n` at the beginning for some reason

Comment (gssbzn) [reply tools/releaser/scripts/update-version.sh:22]: that seems to be a problem of the releaser not of this script

Comment (wtrocki) [reply tools/releaser/scripts/update-version.sh:22]: Closing PR then. Sorry for confusion

**Pull request closed without merging**